Xamarin.Forms.Platform.Android/Platform.cs

```diff
@@ -850,21 +850,24 @@ void SetActionBarTextColor()
 			if(Forms.IsLollipopOrNewer)
 			{
 				int actionbarId = _context.Resources.GetIdentifier("action_bar", "id", "android");
-				if(actionbarId > 0)
+				if (actionbarId > 0)
 				{
-					Toolbar toolbar = (Toolbar)((Activity)_context).FindViewById(actionbarId);
-					
-					for( int i = 0; i < toolbar.ChildCount; i++ )
+					var toolbar = (ViewGroup)((Activity)_context).FindViewById(actionbarId);
+					if (toolbar != null)
 					{
-						if( toolbar.GetChildAt(i) is TextView )
+						for (int i = 0; i < toolbar.ChildCount; i++)
 						{
-							actionBarTitleTextView = (TextView)toolbar.GetChildAt(i);
-							break;
+							if (toolbar.GetChildAt(i) is TextView)
```

Comment:
`GetChildAt(i)` is being called twice. Maybe use c#7:
`GetChildAt(i) is TextView textView` and use that in the inner block.